src/index.js

```diff
@@ -0,0 +1,78 @@
+import bip39 from 'bip39';
+import parseAlgorithm from './algorithm';
+import { composePrivateKey, composePublicKey } from 'crypto-key-composer';
+
+const getFormats = (privateKeyFormat = 'pkcs8-pem') => {
+    const formats = {
+        privateKey: privateKeyFormat,
+        publicKey: privateKeyFormat,
+    };
+
+    if (privateKeyFormat.includes('pem')) {
+        formats.publicKey = 'spki-pem';
+    } else if (privateKeyFormat.includes('der')) {
+        formats.publicKey = 'spki-der';
+    }
+
+    return formats;
+};
+
+const composeKeys = ({ privateKey, publicKey }, keyAlgorithm, options = {}) => {
+    const { format, encryptionAlgorithm, password } = options;
```

Comment:
It's safer to do `options = { ...options, format: 'pkcs8-pem', encryptionAlgorithm: null, password: null }`. The reason is that this will fail if I pass `null` as the options

Reply:
I think I will change it to 
```js
const { format, encryptionAlgorithm, password } = { ...options };
```